src/main/resources/db/migration/V3__add_job_run_definitions.sql

```diff
@@ -1,9 +1,21 @@
 ALTER TABLE jobs DROP COLUMN nominal_time;
+ALTER TABLE jobs ALTER COLUMN description DROP NOT NULL;
+ALTER TABLE jobs ADD COLUMN current_owner_name VARCHAR(64);
+
+ALTER TABLE job_versions ADD COLUMN version UUID;
+ALTER TABLE job_versions ADD COLUMN uri VARCHAR(256);
+ALTER TABLE job_versions DROP COLUMN git_repo_uri;
+ALTER TABLE job_versions DROP COLUMN git_sha;
+ALTER TABLE job_versions ALTER COLUMN input_dataset DROP NOT NULL;
+ALTER TABLE job_versions ALTER COLUMN output_dataset DROP NOT NULL;
 
 CREATE TABLE IF NOT EXISTS job_run_definitions (
     guid UUID PRIMARY KEY,
     job_version_guid UUID REFERENCES job_versions(guid),
     run_args_json TEXT,
-    nominal_time TIMESTAMP WITH TIME ZONE NOT NULL,
+    content_hash UUID NOT NULL,
+    nominal_start_time INTEGER,
+    nominal_end_time INTEGER,
```

Comment:
I probably should have spotted this during review, but I thought we were not allowing changes to .sql files after they were already checked in?

Also, why are the nominal start and end times integers, while created_at is a timestamp? Seems like an unnecessary inconsistency.